api/spec/controllers/mno_enterprise/auth/confirmation_controller_spec.rb

```diff
@@ -33,17 +33,12 @@ module MnoEnterprise
 
       context 'confirmed user' do
         let(:user) { confirmed_user }
-
         context 'with a new email' do
           let(:email) { 'unconfirmed@example.com' }
-
-          before do
+          let!(:api_stubs) do
```

Comment:
I had issue with the stub beeing called twice, also the problem is that when stub_api_v2 is being called, it is the current state of the input that will be saved as json and stored in the stub. Therefore, is the input is changed in another block or let(), the stub won't be updated. It's not dynamic as the argument is saved serialized in the stub.